Formula/faust.rb

```diff
@@ -0,0 +1,63 @@
+class Faust < Formula
+  desc "Functional AUdio STream is language for signal processing and synthesis."
+  homepage "http://faust.grame.fr"
+  option "with-faust0", "install faust 0.x.x not faust 2.x.x.(don't require llvm and openssl.)"
+  option "without-httpd", "don't compile httpdlib (requires GNU libmicrohttpd)"
+  option "without-dynamic", "don't compile httpd & osc supports as dynamic libraries"
+  option "without-sound2faust", "don't compile sound to DSP file converter"
+  option "without-test", "don't install test suite"
+
+  depends_on "pkg-config" => :run
+
+
+  if build.without? "faust0"
+    depends_on "llvm@3.8" => :run
```

Comment:
HEAD version is compatible with `llvm` but a stable release isn't, just because Makefile is only compatible with up to 4.0.0... I'll update it to 'llvm' when next version is released.